### ISSUE TITLE
Questions about codelab

### ISSUE THREAD

**Issue by OLmmyT** [openthread/ot-docs#33]
I have completed this codelab: https://openthread.io/codelabs/openthread-border-router#0

Can you please point me:
- How to see the list of end devices in the Thread network? Shall I use "sudo service otbr-agent status" on the border router?
- How to disconnect the end device (I want to redo the operation but now with the open thread commissioning app).

Thank you!
Regards

**Comment by wgtdkp** [comment]
> How to see the list of end devices in the Thread network? Shall I use "sudo service otbr-agent status" on the border router?

You can view the network topology with OTBR web: https://openthread.io/guides/border-router/web-gui.

> How to disconnect the end device (I want to redo the operation but now with the open thread commissioning app).

you can run command `factoryreset` on the end device to factory reset it.

**Comment by OLmmyT** [comment]
Thanks.
In the meantime I have been able to use the Border Router Web GUI:
In Android Service Browser, I have clicked on _meshcop._udp service, copied the IP address (192.168.43.15 in my case) and pasted it in Chrome (still on Android phone). I'm able to display the Border Router Web GUI and to see the topology.

> you can run command factoryreset on the end device to factory reset it.
I have used "thread stop". If I use the command "factoryreset", will the end device forget all the dataset corresponding to the border router?
